### PR TITLE
Simplifying the localization process

### PR DESCRIPTION
**Pull request by me-shaon**
Tried to simplify the localization process based on the Issue #23.

### CODE REVIEW

**Comment by BenSampo** [comment]
Hey @me-shaon, thanks for the contribution! This is looking good. Would you be able to add some tests to verify that the getDescription method correctly pulls values from the localisation files?

**Comment by me-shaon** [comment]
I was thinking about writing Test but I'm not much experienced about it. I am not sure how to mock the localization files of Laravel to Test the functionality. Could you please guide me about it? :)

**Comment by BenSampo** [comment]
Having looked very briefly you should be able to use the `Illuminate\Translation\Translator` class to load a new lang file which you'll need to create in the test directory.

**Comment by me-shaon** [comment]
@BenSampo I've searched a bit and went through some existing laravel packages Tests to see how they handle the Tests for Localizations. I've added some Tests based on which seems the way to go.

File: tests/EnumLocalizationTest.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ public function testEnumGetDescriptionWithLocalization()
         $this->assertEquals('Súper administrador', UserTypeWithLocale::getDescription(UserType::SuperAdministrator));
     }
 
-    public function testEnumGetDescriptionForMissingLocalizationKey()
+    public function test_enum_get_description_for_missing_localization_key()
```
Comment (me-shaon) [inline tests/EnumLocalizationTest.php]: Is it possible to maintain 'CamelCase' in the test method names? I think it's kind of a convention.

Comment (BenSampo) [reply tests/EnumLocalizationTest.php]: Snake case is much easier to read, especially for longer test names

Comment (me-shaon) [reply tests/EnumLocalizationTest.php]: That's true. 👍

**Comment by me-shaon** [comment]
@BenSampo yup, I was going through the recent changes you've made and it seems very clean approach to me too. Appreciate the job you did here. 👍

**Comment by BenSampo** [comment]
Merged and tagged ✅

**Pull request merged**